go.mod

```diff
@@ -209,5 +209,5 @@ replace (
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.2
 
 	// need the modifications eksa made to the capi api structs
-	sigs.k8s.io/cluster-api => github.com/abhay-krishna/cluster-api v1.2.0-custom.rc2
+	sigs.k8s.io/cluster-api => github.com/jiayiwang7/cluster-api v0.0.0-20221123142628-4b8c7b3f61eb
```

Comment:
discard this change when https://github.com/abhay-krishna/cluster-api/pull/3 is merged